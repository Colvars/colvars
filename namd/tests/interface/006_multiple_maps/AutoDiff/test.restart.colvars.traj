# step         multimap               map                    map2                   E_harmonic1          
          20    1.32181314890177e+00   7.88136398571338e-01   2.66838375165217e-01   5.74921756253162e+02
          21    1.32333304934533e+00   7.87083765319052e-01   2.68124642013139e-01   5.72347702635361e+02
          22    1.32499770184017e+00   7.86008176990984e-01   2.69494762424593e-01   5.69535128151316e+02
          23    1.32681113971906e+00   7.84921848718747e-01   2.70944645500155e-01   5.66479052007945e+02
          24    1.32877492062567e+00   7.83832767412288e-01   2.72471076606692e-01   5.63178883976340e+02
          25    1.33088800309788e+00   7.82745511963395e-01   2.74071245567243e-01   5.59638580497928e+02
          26    1.33314680516952e+00   7.81662611141694e-01   2.75742097013915e-01   5.55866479319517e+02
          27    1.33554555238588e+00   7.80585412943197e-01   2.77480069721342e-01   5.51874641192731e+02
          28    1.33807272031409e+00   7.79513550871742e-01   2.79279584721176e-01   5.47684654490479e+02
          29    1.34070995934557e+00   7.78445814092469e-01   2.81132072626551e-01   5.43329197132649e+02
          30    1.34343234254707e+00   7.77380758949593e-01   2.83025791798738e-01   5.38851361016535e+02
          31    1.34621265955383e+00   7.76317092898466e-01   2.84947783327683e-01   5.34297358159594e+02
          32    1.34902320439734e+00   7.75253763168172e-01   2.86884720614585e-01   5.29713485516381e+02
          33    1.35183835587595e+00   7.74190062372004e-01   2.88824146751975e-01   5.25141896141982e+02
          34    1.35463948333175e+00   7.73123224330681e-01   2.90758129500534e-01   5.20612745592890e+02
          35    1.35741424816319e+00   7.72049870092216e-01   2.92682189035486e-01   5.16145560579601e+02
          36    1.36015216001605e+00   7.70969705192076e-01   2.94591227411985e-01   5.11756572915164e+02
          37    1.36284869717449e+00   7.69882935711069e-01   2.96482880731709e-01   5.07452228365309e+02
          38    1.36550660095620e+00   7.68790124370535e-01   2.98358238292834e-01   5.03227341787687e+02
          39    1.36813638662645e+00   7.67687594578605e-01   3.00224396023923e-01   4.99064532381845e+02
          40    1.37074524080494e+00   7.66575968488260e-01   3.02084636158342e-01   4.94951939962034e+02
