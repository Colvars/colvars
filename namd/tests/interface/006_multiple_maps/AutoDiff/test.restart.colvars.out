colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: This version was built with the C++11 standard or higher.
colvars: Summary of compile-time features available in this build:
colvars:   - SMP parallelism: not available
colvars:   - Lepton custom functions: available
colvars:   - Tcl interpreter: available
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "multimap"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "map"
colvars:     # componentCoeff = 0.0034027
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map"
colvars:     # mapID = -1 [default]
colvars:     # atomWeights =  [default]
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "maprot"
colvars:     # componentCoeff = 0.0068054
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map2"
colvars:     # mapID = -1 [default]
colvars:     # atomWeights =  [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.02
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0.02 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "map"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "map"
colvars:     # componentCoeff = 0.0034027
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map"
colvars:     # mapID = -1 [default]
colvars:     # atomWeights =  [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "map2"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "map2"
colvars:     # componentCoeff = 0.0034027
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map2"
colvars:     # mapID = -1 [default]
colvars:     # atomWeights =  [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 3 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { multimap }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 1.3 }
colvars:   # targetCenters = { 2 }
colvars:   Initial step for restraint change: 0
colvars:   # targetNumSteps = 20
colvars:   # targetNumStages = 0 [default]
colvars:   # outputAccumulatedWork = off [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 1
colvars:   # decoupling = off [default]
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "multimap" will be rescaled to 2500 according to the specified width (0.02).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (2 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Harmonic colvar bias implementation:
colvars: - Linear and polynomial combination of colvar components:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - Volumetric map-based collective variables:
colvars: - mapTotal colvar component:
colvars:   Fiorin2020 https://doi.org/10.1002/jcc.26075
colvars: 
colvars: - NAMD engine:
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: - GridForces volumetric map implementation for NAMD:
colvars:   Wells2007 https://doi.org/10.1063/1.2770738
colvars: 
colvars: updating target temperature (T = 0 K).
colvars: ----------------------------------------------------------------------
colvars: Loading state from text file "test.colvars.state".
colvars:   Restarting collective variable "multimap" from value: 1.32181
colvars:   Restarting collective variable "map" from value: 0.788136
colvars:   Restarting collective variable "map2" from value: 0.266838
colvars:   Restarted harmonic bias "harmonic1" with step number 20.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (2 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: Current simulation parameters: initial step = 20, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
